chevah/compat/tests/elevated/test_system_users.py

```diff
@@ -423,8 +423,10 @@ def test_executeAsUser_Unix(self):
             self.assertEqual(TEST_ACCOUNT_GROUP, impersonated_groupname)
             self.assertNotEqual(initial_uid, uid)
             self.assertNotEqual(initial_gid, gid)
-            self.assertNotEqual(initial_groups, impersonated_groups)
+            # FIXME:3808:
+            # Investigate why this no longer works/passes on OSX.
```

Comment:
I'm against fixing this here for the following reasons: I don't know what is causing this but master is also affected; I prefer to have a clean PR and not mix group related bugs with impersonation.